manifests/server/foreman.pp

```diff
@@ -0,0 +1,33 @@
+# @summary Set up Foreman integration
+# @api private
+class puppet::server::foreman (
+  Boolean $katello = false,
+) {
+  if $katello {
+    include certs::puppet
```

Comment:
Does this imply https://github.com/theforeman/puppet-puppet/blob/master/metadata.json#L17 now needs to require `katello-certs` or do we allow 'implied' or 'if present' dependencies?

Reply:
We already have puppet-puppetserver_foreman as a soft dependency: https://github.com/theforeman/puppet-puppet#foreman-integration